tests/resources/settings.py

```diff
@@ -0,0 +1,198 @@
+# Copyright The IETF Trust 2019, All Rights Reserved
+# Copyright 2015 Cisco and its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Django settings for yangvalidator project.
+
+Generated by 'django-admin startproject' using Django 2.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+__author__ = "Miroslav Kovac, Carl Moberg"
```

Comment:
with new files please get us out as authors and put your name only same with email. and copyright must have current year no Cisco anymore this is only IETF project. we are adding IETF to files were cisco was but new files contain only IETF

Reply:
Most of the file was copied over and unchanged, so I think the cisco copyright notice should be kept.